starter/level-three.js

```diff
@@ -2,3 +2,30 @@
  * DOTS: Level Three
  *
  */
+//variables
+let scoreDisplay = document.querySelector(".js-score");
+let ball = document.querySelectorAll(".ball");
+let levelWinner = document.querySelector(".level-winner");
+let score = 0;
+let scoreAdd = 10;
+
+
+//event listener for ball click
+const elements = document.querySelectorAll('.js-ball');
+for (let i = 0; i < elements.length; i++) {
+  elements[i].addEventListener('click', function() {
```

Comment:
querySelectorAll allows you to run a forEach helper method, in the case where you need to loop 👍